apiserver/test-apiserver.sh

```diff
@@ -18,12 +19,42 @@ export PYTHONPATH=$(ls $SPARK_HOME/python/lib/py4j-*-src.zip):$SPARK_HOME/python
 export JAR=../hail/build/libs/hail-all-spark.jar
 export PYSPARK_SUBMIT_ARGS="--conf spark.driver.extraClassPath=$JAR --conf spark.executor.extraClassPath=$JAR pyspark-shell"
 
-python3 apiserver/apiserver.py &
+HAIL_JWT_SECRET_KEY_FILE=jwt-test-secret-key python3 apiserver/apiserver.py &
 server_pid=$!
 
 ../until-with-fuel 30 curl -fL http://localhost:5000/healthcheck
 
-export HAIL_TEST_SERVICE_BACKEND_URL=http://localhost:5000
```

Comment:
@cseed in particular, this was wrong

Reply:
I added `test/test_apiserver.py` which verifies the backend

Reply:
I don’t think that’s checked in

Reply:
How was this wrong?

Reply:
This is used in test/hail/helpers, in the setupModule. I don’t think it’s wrong

Reply:
@cseed it looked to me like the variable name was out of date? Hail’s Backend.py looks for HAIL_APISERVER_URL, I think

Reply:
Yes you’re right, there is a separate url in tests. When I initially commented about the test I thought that you had changed this, and that the changes hadn’t been checked in.

Reply:
Ah, yes, there are two variables, one for tests, and one in general (in context.py).  I probably put the test one in before the general one is in place.  With this change you should remove it from helpers.py.